python/tritonserver/_api/_model.py

```diff
@@ -128,9 +127,7 @@ def create_request(self, **kwargs: Unpack[InferenceRequest]) -> InferenceRequest
         _server=<tritonserver._c.triton_bindings.TRITONSERVER_Server
         object at 0x7f5827156bf0>, request_id=None, flags=0,
         correlation_id=None, priority=0, timeout=0, inputs={},
-        parameters={}, output_memory_type=None,
-        output_memory_allocator=None, response_queue=None,
-        _serialized_inputs={})
+        parameters={}, output_memory_type=None, _serialized_inputs={})
```

Comment:
Todo: for the future - we should find a way to document how to add a custom allocator - memory allocation is an area where customization is probably needed

Reply:
There is a story DLIS-7824 for supporting GPU memory allocation, but not a custom allocator that can be supplied by the user.

do you think that will be enough? or there will be use-case where there are memory types that we cannot implement in advance and it has to be supplied by the user at runtime?